addon/templates/components/course-materials.hbs

```diff
@@ -111,8 +158,14 @@
               {{/if}}
             </td>
           </tr>
+        {{else}}
+          <tr>
+            <td colspan="9" align="center">
+              {{if this.sessionQuery "No Results Found" "No Session Learning Materials Available"}}
```

Comment:
same here, these messages need to be translatable.

Reply:
Fixed.